go.mod

```diff
@@ -78,8 +78,8 @@ require (
 	k8s.io/utils v0.0.0-20211116205334-6203023598ed
 	kubevirt.io/api v0.0.0-00010101000000-000000000000
 	kubevirt.io/client-go v0.0.0-00010101000000-000000000000
-	kubevirt.io/containerized-data-importer v1.41.0
-	kubevirt.io/containerized-data-importer-api v1.50.0
+	kubevirt.io/containerized-data-importer v1.55.0
```

Comment:
Where do we need the cdi (not the API)?

Reply:
Because kubevirt uses one or two functions from CDI (to determine some fields) to ensure CDI and kubevirt use the same calculation.